redundant-bmc/src/rbmc_tool.cpp

```diff
@@ -258,13 +292,32 @@ int main(int argc, char** argv)
     bool info{};
     bool extended{};
     bool resetSibling{};
+    bool disableRedundancy{};
+    bool enableRedundancy{};
     sdbusplus::async::context ctx;
 
-    auto* flag = app.add_flag("-d", info, "Display RBMC Information");
-    app.add_flag("-e", extended, "Add extended RBMC details to the display")
+    auto* displayGroup = app.add_option_group("Display RBMC information");
+    auto* flag =
+        displayGroup->add_flag("-d", info, "Display basic RBMC information");
```

Comment:
Does the the -d shows if the override is being used to enable/disable redundancy? If not can we add that (maybe under -e) so we know the value of the override?